src/main/drivers/serial_pinconfig.c

```diff
@@ -247,5 +241,33 @@ void pgResetFn_serialPinConfig(serialPinConfig_t *serialPinConfig)
         serialPinConfig->ioTagInverter[SERIAL_PORT_IDENTIFIER_TO_INDEX(defpin->ident)] = defpin->inverterIO;
     }
 }
+
+#if defined(USE_SOFTSERIAL1) || defined(USE_SOFTSERIAL2)
+typedef struct softSerialDefaultPin_s {
+    serialPortIdentifier_e ident;
+    ioTag_t rxIO, txIO;
+} softSerialDefaultPin_t;
+
+static const softSerialDefaultPin_t softSerialDefaultPin[SOFTSERIAL_COUNT] = {
+#ifdef USE_SOFTSERIAL1
+    { SERIAL_PORT_SOFTSERIAL1, IO_TAG(SOFTSERIAL1_RX_PIN), IO_TAG(SOFTSERIAL1_TX_PIN) },
+#endif
+#ifdef USE_SOFTSERIAL2
+    { SERIAL_PORT_SOFTSERIAL2, IO_TAG(SOFTSERIAL2_RX_PIN), IO_TAG(SOFTSERIAL2_TX_PIN) },
+#endif
+};
+
+PG_REGISTER_WITH_RESET_FN(softSerialPinConfig_t, softSerialPinConfig, PG_SOFTSERIAL_PIN_CONFIG, 0);
+
+void pgResetFn_softSerialPinConfig(softSerialPinConfig_t *softSerialPinConfig)
+{
+    for (size_t index = 0 ; index < ARRAYLEN(softSerialDefaultPin) ; index++) {
+        const softSerialDefaultPin_t *defpin = &softSerialDefaultPin[index];
+        softSerialPinConfig->ioTagRx[SOFTSERIAL_PORT_IDENTIFIER_TO_INDEX(defpin->ident)] = defpin->rxIO;
+        softSerialPinConfig->ioTagTx[SOFTSERIAL_PORT_IDENTIFIER_TO_INDEX(defpin->ident)] = defpin->txIO;
+    }
+}
```

Comment:
New parameter groups now go into the [../src/main/pg/](https://github.com/betaflight/betaflight/tree/master/src/main/pg) folder.

You can have a look at [my PR for reference](https://github.com/betaflight/betaflight/pull/11765/commits) where I do the refactoring for the RPM Filter PG.